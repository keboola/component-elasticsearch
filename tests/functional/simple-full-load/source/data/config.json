{
  "parameters": {
    "index_name": "myindex",
    "primary_keys": ["id"],
    "query": "",
    "date": {
      "shift": "yesterday",
      "format": "%Y-%m-%d",
      "time_zone": "UTC"
    },
    "scheme": "http",
    "incremental": true,
    "storage_table": "test",
    "db": {
      "hostname": "elasticsearch",
      "port": 9200
    },
    "authentication": {
      "auth_type": "basic",
      "username": "elastic",
      "#password": "root"
    },
    "debug": false
  }
}