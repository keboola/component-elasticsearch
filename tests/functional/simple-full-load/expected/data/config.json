{
  "parameters": {
    "index_name": "myindex",
    "primary_keys": ["id"],
    "query": "",
    "date": {
      "shift": "yesterday",
      "format": "%Y-%m-%d",
      "time_zone": "UTC"
    },
    "scheme": "http",
    "incremental": true,
    "storage_table": "search_analytics",
    "db": {
      "hostname": "host.docker.internal",
      "port": 9200
    },
    "authentication": {
      "auth_type": "basic",
      "username": "elastic",
      "#password": "root"
    },
    "debug": false
  }
}